### PR TITLE
cf-mgmt 1.2.0 (new formula)

### PR DESCRIPTION
**Pull request by neilotoole**
Initial commit of a formula for the cf-mgmt tool for the HP Helion Development Platform. This is a tool for creating, managing, updating, and deleting ALS (CloudFoundry) clusters.

### CODE REVIEW

File: cf-mgmt.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  url "http://clients.als.hpcloud.com/cf-mgmt-1.2.0-osx-x86_64.zip"
+  version "1.2.0"
+  sha256 "f3992859b6b8b97900d3b6cce8e6d087f64561d82eaded9d6ea43179909b52d1"
+
+  def install
+    bin.install "cf-mgmt"
+  end
+
+  test do
+    system "#{bin}/cf-mgmt", "--version"
```
Comment (mistydemeo) [inline cf-mgmt.rb]: Is it possible to write a test that tests the functionality of this?

Comment (neilotoole) [inline cf-mgmt.rb]: Given that this is a client for creating/managing cloud services, it really can't be tested sensibly without a cloud instance to test against, and that would obviously require credentials, a permanent instance, etc, which is not really feasible.

**Comment by neilotoole** [comment]
@mistydemeo  Any chance of moving this package forward?

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by bfontaine** [comment]
@BrewTestBot test this please.

**Comment by DomT4** [comment]
Any interest here still?

**Comment by DomT4** [comment]
This should be submitted to https://github.com/caskroom/homebrew-cask now. We're in the process of deprecating this repository.

We'll take existing formula version bumps here whilst migration is ongoing, but we don't really want to add new formulae, apologies. Thank you for the submission to Homebrew nonetheless ❤️.

**Pull request closed without merging**